flybywiresim/a32nx
# build: use mach for instruments build

## Summary of Changes

This switches the build pipeline for building instruments to use https://github.com/Synaptic-Simulations/mach/ instead of rollup and a custom plugin.

Included are also fixes for some issues that trip `esbuild` up.

* [x] Introduce new build system
* [x] Remove old build system (keep ace build scripts for now) 

## Testing instructions


* Make sure ALL instruments work properly
* Make sure no negative performance difference exists compared to dev


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
